src/script/components/message/CallMessage.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+import NamedIcon from 'Components/NamedIcon';
+import {CallMessage as CallMessageEntity} from '../../entity/message/CallMessage';
+
+import {registerReactComponent} from 'Util/ComponentUtil';
+
+export interface CallMessageProps {
+  message: CallMessageEntity;
+}
+
+const CallMessage: React.FC<CallMessageProps> = ({message}) => {
+  const isCompleted = message.was_completed();
+  const unsafeSenderName = message.unsafeSenderName();
+  const caption = message.caption();
+  const displayTimestampShort = message.displayTimestampShort();
+  const displayTimestampLong = message.displayTimestampLong();
+
+  // Equivalent for `ko.bindingHandlers.showAllTimestamps`
```

Comment:
How about removing the comment and putting it here? https://github.com/wireapp/wire-webapp/wiki/React-refactoring-goals-&-conventions#examples

Reply:
This is actually a thing that needs to be reconsidered when the whole `message.ts` is refactored to react because this is needed in every kind of message but the state is floating outside of the component.

(it makes all timestamps in the conversation visible when hovering just one)